build/registry/s3.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"bytes"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+)
+
+func s3UploadFile(session *session.Session, bucket string, filePath string, key string) error {
+	upFile, err := os.Open(filePath)
+	if err != nil {
+		return err
+	}
+	defer upFile.Close()
+
+	upFileInfo, _ := upFile.Stat()
+	fileSize := upFileInfo.Size()
+	fileBuffer := make([]byte, fileSize)
+	upFile.Read(fileBuffer)
+
+	_, err = s3.New(session).PutObject(&s3.PutObjectInput{
+		Bucket:               aws.String(bucket),
+		Key:                  aws.String(key),
+		Body:                 bytes.NewReader(fileBuffer),
+		ContentLength:        aws.Int64(fileSize),
+		ContentDisposition:   aws.String("attachment"),
+		ServerSideEncryption: aws.String("AES256"),
+	})
```

Comment:
Note for the reviewer: do you know if we need to set additional permissions or ACL when uploading this in the Falco bucket?

Reply:
I don't think so, based on this: https://github.com/falcosecurity/test-infra/blob/be8f54f98393f743a0374b664f36f717c5505fb9/images/build-plugins/build-and-publish.sh#L41